stdlib/unittest/mock.pyi

```diff
@@ -428,14 +428,14 @@ if sys.version_info >= (3, 8):
     class AsyncMock(AsyncMockMixin, AsyncMagicMixin, Mock): ...  # type: ignore # argument disparities between base classes
 
 class MagicProxy:
-    name: Any
+    name: str
```

Comment:
I'm not sure if I should change this, however it doesn't really make much sense to me that `name` is typed as `Any` in the module's stubs, when it's clear that it's supposed to be a string (at least from what I could tell from the CPython source).